fastlane/lib/fastlane/actions/setup_ci.rb

```diff
@@ -87,18 +90,20 @@ def self.available_options
                                        default_value: false),
           FastlaneCore::ConfigItem.new(key: :provider,
                                        env_name: "FL_SETUP_CI_PROVIDER",
-                                       description: "CI provider",
+                                       description: "CI provider. If none is set, the provider is detected automatically",
```

Comment:
This is misleading as only circleci would be detected and imho as discussed above, this is outside of the scope of what this PR should do.

Reply:
Please see my explanation above.